espresso/environment/espresso_dev_node_test.go

```diff
@@ -176,6 +178,83 @@ func runSimpleL2Burn(ctx context.Context, t *testing.T, system *e2esys.System) {
 	cancel()
 }
 
+// TestE2eDevNetWithEspressoSimpleTransactions launches the e2e Dev Net with the Espresso Dev Node
+// and runs a couple of simple transactions to it.
+func TestE2eDevNetWithInvalidAttestation(t *testing.T) {
```

Comment:
Could you create a new file named _5_batch_authentication_test.go_ and put the new tests there?